ui/imports/shared/stores/CommunityTokensStore.qml

```diff
@@ -153,7 +153,7 @@ QtObject {
             root.setSignerStateChanged(communityId, communityName, status, url)
         }
 
-        function onOwnershipLost(communityId, communityName) {
+        function onOwnershipNodeLost(communityId, communityName) {
```

Comment:
How is this related? I see that it fixes something but you don't mention it anywhere. Put it at least in a separate commit

Reply:
This error was logged in the console, just around the time the error with avatar has occurred. Its a real issue, but indeed not related to the avatar and ring bugs.
I'll put it in a separate commit.

Reply:
Yeah, this is good, here's the NIM signal:
```nim
proc ownershipNodeLost*(self: View, communityId: string, communityName: string) {.signal.}
```

Reply:
yes, I've found that signal on nim side and rename handler in qml accordingly